### ISSUE TITLE
Relative path is not getting resolved in Marko On Windows environment

### ISSUE THREAD

**Issue by mrrajesh1982** [marko-js/marko#158]
Hi Patrick,

We are using Marko JS on our application. Below is the GitHub URL:

https://github.corp.ebay.com/OCS-Dev/webforms/tree/rti

One of our component we provide dependency: 
https://github.corp.ebay.com/OCS-Dev/webforms/blob/rti/src/components/email-form/browser.json

{
  "dependencies": [
    "require: ./index",
    "src/ds3/browser.json",
    "src/third-party/jquery.js",
    "require: src/components/call-me",
    "require: src/components/submit-button"
  ]
}

Relative path is not getting resolved in On Windows environment(working expected on MAC though). I see below exception in browser console when I hit our page: 

http:/localhost:8080/email/compose/AQABAAAC0CtGM/B68cFDXP472Z5EMUy3itN2JRaj564KXegqAHIAcDOylgBccNyhrWzcCHXX8kyHd81d6zeVrAZ5NEj0cUgs9X3aPVuBb0a7nf45lhmYUUZQTFGsjCbg408mqCvMT4NheESGWhArH+yzWgabsk4+fav0SAH3OpnVL5D2FBxkkBKbKx/7aa/AmDwMgronfRfDBJQued354L337awe7pBXlXHA6BO7ThumDHPch6n6qwytqQ4oEnaKoCcCgYO/d+/5TjiCWbo3FhQNzjicgj1z6Zm7OX4RkbfY5nJpdixp3JJzaKLtmyxp24cLy1SY+8ckVqdOvEPfPJyspLxkixDHQXqRyEi7/vJ7dAhbPACJ8Msr6bfMxg+geA15ei7D0n+JY5sMnqm/IM7A7oH5hvmhs0p3DHRzLcNPNOB9ieI1Em0KpQaTzyLdP+7XKK9ExgVSkK+IRBrey6NXwf4gFo3KJsDkzKYUJ/qMZjZdsAwpxXmRTLbJzMx8CqxEVz54R/H6OQTOHHksx1Jnyq9UzuNw/1UVFa3NkdDvVnN/mHtBeMWrJmWBqpMhzAoXsgi5btfDRIj6TLSKDQP4/7h5diDcRtdIt3ZRsOl6KvabSx8X0HnAljzE0M7AzO2BK50+76RtzAor2mc4wqv/TYYfk5EOfoXTIP5Xu2i+oRDP8EnaYR2pM5jaJn8m4V8Iz3XHt1Cr7qEuCep4q0gKUtxIRrXSxoqYKzJtbjryfnEkNTDXVX2YirAyeOtITN1QJHtv/GPaE09NGCcxvuJz51fPkBpVWa9WS5YaYXo5px6svVnCXWt8ip+yhhDjA28Hgepjy9oqoMaexfKBtnecoOJp0+8/xJPWUT6RlZDnMMraDgCfqBfHtuVnl5kMbZbAmuWVKI2n+ddhg/GaT23NQ6IATuv3XoZ1KCJu9JKgIbo9ULEgb1fnQVnYmDWPuVXAA5qIDg==

Cannot find module "..\call-me" from "/src/components/email-form"

Basically the generated Marko.js file creates the dependencies like below: (template.marko.js) 

  ___call_me_index_js = __renderer(require("..\\call-me")),
      ___submit_button_index_js = __renderer(require("..\\submit-button"));

Any pointer would be helpful.

Thanks
Rajesh

**Comment by patrick-steele-idem** [comment]
I believe this issue has been resolved with improvements to the [deresolve](https://github.com/raptorjs/raptor-modules/blob/04fc6554b800277e1c62b4e1d66586bfb45acf90/resolver/lib/deresolve.js) function. If I am mistaken, please reopen the issue. Thanks.
